tests/foreman/ui/test_activationkey.py

```diff
@@ -102,10 +102,10 @@ def test_positive_end_to_end_crud(session, module_org):
         # Update activation key with new name
         session.activationkey.update(name, {'details.name': new_name})
         assert session.activationkey.search(new_name)[0]['Name'] == new_name
-        assert not session.activationkey.search(name)
+        assert session.activationkey.search(name)[0]['Name'] != name
```

Comment:
I wonder if we need this assertion at all. If update failed, then search by `new_name` will return empty table and previous assertion will fail.

This only checks that update didn't result in creating new entity instead of updating old, which sounds like strange thing to test for. It's not like we ever had such issue before.

Reply:
I guess it's really up to component owner to decide. Feel free to dismiss that comment, I'm just thinking aloud.

Reply:
I think these tests were added by Alex earlier, not by any component owner. I don't think so it is a blocking issue, we can still merge this PR with change.